src/components/motebehov/MeldingTilArbeidsgiver.tsx

```diff
@@ -16,7 +16,6 @@ interface MeldingTilArbeidsgiverProps {
 export const MeldingTilArbeidsgiver = ({
   sykmelding,
 }: MeldingTilArbeidsgiverProps) => {
-  const innspillTilArbeidsgiver = sykmelding.innspillTilArbeidsgiver;
   const skalVise = erMeldingTilArbeidsgiverInformasjon(sykmelding);
```

Comment:
Kan evnt erstatte skalVise med inlinet sykmelding.innspillTilArbeidsgiver. Da blir den castet til boolean og man får samme truthy/falsy sjekk.

Litt smak og behag, men slipper både variabler og util-metode.

Reply:
Ja, jeg vurderte det 🤔 Tanken var at det er util-metoden som bestemmer om vi viser prikker på sykmeldingen, så hvis metoden endres, vet vi at det er samme regler som gjelder for visning av prikker og tekst. I praksis blir resultatet det samme sånn som det er nå da 👍

Reply:
Skjønner :)